indexer/src/main/java/au/org/aodn/esindexer/utils/GeometryUtils.java

```diff
@@ -44,7 +44,7 @@ public enum PointOrientation {
 
     @Getter
     @Setter
-    protected static double coastalPrecision = 0.05;
+    protected static double coastalPrecision = 0.1;
```

Comment:
should we have a comment to explain why that value or simply a random pick?

Reply:
Added comment